tests/vk-regression/vk-regression.json

```diff
@@ -288,16 +296,29 @@
     }
   },
   "sha256": {
-    "digest": "38eaa6264b1e4c1c0b5787c06efd92aa7855890aeeafed91818d466f1b109ca5",
+    "digest": "2502d4e6456fcb0b405f2bf995b34bd1d703951bcd0d461247327d3c86335691",
```

Comment:
oh! This shouldn't have changed, let's investigate

Reply:
Now, this doesn't change.